packages/store/tests/execution/dispatch-outside-zone-ngxs-execution-strategy.spec.ts

```diff
@@ -295,62 +285,89 @@ describe('DispatchOutsideZoneNgxsExecutionStrategy', () => {
     });
   });
 
-  xit('should warn if zone is "nooped"', async(() => {
-    @State({ name: 'foo' })
-    class FooState {}
+  it(
+    'should warn if zone is "nooped"',
+    freshPlatform(async () => {
+      // Arrange
+      @State({ name: 'foo' })
+      class FooState {}
 
-    @Component({
-      selector: 'app-root',
-      template: ''
-    })
-    class MockComponent {}
+      @Component({
+        selector: 'app-root',
+        template: ''
+      })
+      class MockComponent {}
 
-    @NgModule({
-      imports: [
-        BrowserModule,
-        NgxsModule.forRoot([FooState], {
-          executionStrategy: DispatchOutsideZoneNgxsExecutionStrategy
-        })
-      ],
-      declarations: [MockComponent],
-      entryComponents: [MockComponent]
+      @NgModule({
+        imports: [
+          BrowserModule,
+          NgxsModule.forRoot([FooState], {
+            executionStrategy: DispatchOutsideZoneNgxsExecutionStrategy
+          })
+        ],
+        declarations: [MockComponent],
+        bootstrap: [MockComponent]
+      })
+      class MockModule {}
+
+      // Act
+      const warnings: string[] = [];
+
+      console.warn = (...args: string[]) => {
+        warnings.push(args[0]);
+      };
```

Comment:
This should be restored back to the normal console.warn afterwards, otherwise this test could cause unexpected results.
This can be done in a `finally`.

Reply:
Potentially you could make a helper method similar to `freshPlatform` called `preserveConsole` that just stores a copy of all the properties of console to another object (`const backupOfConsole = { ...console };`) then it executes the test, and then in a `finally` it writes all those properties back to console (`Object.assign(console, backupOfConsole)`);
... or something like that.